src/Lykke.SettingsReader/SettingsProcessor/Checkers/AmqpChecker.cs

```diff
@@ -0,0 +1,32 @@
+﻿using System;
+using System.Reflection;
+using Lykke.SettingsReader.Exceptions;
+using Lykke.SettingsReader.Extensions;
+using Lykke.SettingsReader.Helpers;
+
+namespace Lykke.SettingsReader.Checkers
+{
+    public class AmqpChecker : ISettingsFieldChecker
+    {
+        public CheckFieldResult CheckField(object model, PropertyInfo property, object value)
+        {
+            var val = value.ToString();
+
+            if (val.SplitParts('@', 2, out var values) && values[1].SplitParts(':', 2, out var amqpValues))
+            {
+                string address = amqpValues[0];
+
+                if (!int.TryParse(amqpValues[1], out var port))
+                    throw new CheckFieldException(property.Name, value, "Invalid port");
+
+                return new CheckFieldResult
```

Comment:
Maybe here we can check Connecting to a Broker by amqp protocol
https://www.rabbitmq.com/dotnet-api-guide.html
it's goog for ckeck user, password, tcp